molten-http-client/src/test/java/com/hotels/molten/http/client/OkHttpTest.java

```diff
@@ -62,6 +64,7 @@ public void skeletonToTestOkHttpDirectly() throws InterruptedException {
         ConnectionPool connectionPool = connectionPoolFor(concurrency);
         Dispatcher dispatcher = dispatcherFor(concurrency);
         OkHttpClient.Builder clientBuilder = new OkHttpClient().newBuilder()
+                .protocols(Collections.singletonList(Protocol.H2_PRIOR_KNOWLEDGE))
```

Comment:
Mind the indentation. On the other hand, you could use `java.util.List#of(E)` instead.

Reply:
From the comment of Adam, I removed this.